Cargo.toml

```diff
@@ -11,10 +11,10 @@ version = "0.17.0"
 edition = "2018"
 
 [dependencies]
-tari_utilities = { version = "0.5", features = ["zero", "std"] }
+tari_utilities = { package = "tari_utilities", version = "0.5.1", features = ["zero", "std"] }
```

Comment:
Do you need the `package` specification here?

Reply:
Not necessary, will remove